packages/mobile/lib/contexts/dashboard/actions/handleClaimActivity.action.ts

```diff
@@ -0,0 +1,16 @@
+import { isStrongholdUnlocked } from '@core/profile-manager'
+import { Activity, claimActivity } from '@core/wallet'
+import { closeDrawer, DrawerId, openDrawer } from '../../../auxiliary/drawer'
+
+export async function handleClaimActivity(activity: Activity): Promise<void> {
```

Comment:
Wondering if this function belongs in a new `activity` context rather than `dashboard`, I think that might be a little too vague. Alternatively `wallet` would also make sense if you wanted to combine an `activity` and `token` context.

Reply:
moved to wallet, also opened an issue to refactor the dashboard context, it should be wallet